CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 4.10.0
+## Bugfixes
```

Comment:
Missing a section from the `.latest.md`

Reply:
🤦‍♂️

Reply:
@tonidero @vegaro I think there needs to be a change to the _fastlane_ plugin to make these sections in `CHANGELOG`.md use `###` instead of `##` (like in `CHANGELOG.latest.md`

Reply:
Thanks for reporting! I can fix tomorrow morning

Reply:
Done in this PR: https://github.com/RevenueCat/fastlane-plugin-revenuecat_internal/pull/14